### DISCUSSION TITLE
[Suggestion] Add option for sorting ground items in Menu Entry Swapper

### DISCUSSION THREAD

**Discussion by harry-graham** [runelite/runelite#15989] [Q&A] [answered]
It would be very helpful to have some options to help with picking up items more easily by left-click.

### Example

You've just killed a hill giant, which dropped an iron dagger and some big bones.

The default pick-up order is iron dagger first, then big bones. \
Big bones are usually always the last item to be picked up (to my knowledge).

It would be great to have an option to be able to pick up the big bones first, by left-click, without picking up the iron dagger.

Two ideas that might allow this:
a) being able to sort ground items pick-up order by GE price (previously suggested in #1711)
b) being able to move certain items to the top of the pick-up order

### Note

I've seen a similar issue raised in the past (#1711), which was closed and decided not to implement, but it was not fully explained why. The reasoning was:

> We decided to not reorder ground item menus to be able to spam left-click.

It would be great to understand why "spamming left-click" would be an issue, it just sounds like a quality-of-life improvement to me?

**Comment by Adam-** [comment, marked as answer]
You can already change the item pickup order by hiding items you want to deprioritize. We could expand on that I guess to include GE price or something else.

**Reply by harry-graham** [reply]
@Adam- thanks, I didn't know we could do that. Are there instructions anywhere on how to do it?

**Reply by Adam-** [reply]
Enable the "deprioritize menu hidden items" option

**Reply by harry-graham** [reply]
Ah, I didn't realise there was a "Ground Items" plugin, I was looking at the "Menu Entry Swapper". Thanks! 🙌🏼
